src/nft-full/PlaceOfferButton.tsx

```diff
@@ -33,25 +33,12 @@ export const PlaceOfferButton = ({ allowOffer }: PlaceOfferButtonProps) => {
     if (!data) {
       return;
     }
-    if (
-      data.nft.contract.knownContract !== "zora" &&
-      data.pricing.auctionType === AuctionType.RESERVE
-    ) {
-      return [
-        ZORA_SITE_URL_BASE,
-        "collections",
-        data.nft.contract.address,
-        data.nft.tokenId,
-        "auction",
-        "bid",
-      ];
-    }
-
     return [
```

Comment:
This doesn't take into account that non-zora NFTs cannot have offers.
You can add a check 
```
if (data.pricing.auctionType !== AuctionType.RESERVE && data.nft.contract.knownContract !== 'zora') {
  return;
}
```

to prevent that case from rendering

Reply:
Ah ok. Sounds good.